src/weakptr.c

```diff
@@ -636,7 +645,7 @@ static Obj CopyObjWPObj(Obj obj, Int mut)
         ADDR_OBJ(copy)[0] = CONST_ADDR_OBJ(obj)[0];
     }
     else {
-        copy = NewBag( T_PLIST+IMMUTABLE, SIZE_OBJ(obj) );
+        copy = NEW_PLIST(T_PLIST + IMMUTABLE, LengthWPObj(obj));
```

Comment:
```suggestion
        copy = NEW_PLIST_IMM(T_PLIST, LengthWPObj(obj));
```
I was trying in the past to get rid of as many uses of `IMMUTABLE` as possible, in the hope that at some point we turn it into an object flag (also Markus worked hard towards that goal).